pyanaconda/bootloader.py

```diff
@@ -2214,6 +2214,12 @@ def updateNVRAMBootList(self):
         # Remove all other occurances of it.
         boot_list = [boot_disk] + [x for x in boot_list if x != boot_disk]
 
+        # The boot-device NVRAM variable has a maximum number of devices allowed,
+        # don't add more than the limit in the ibm,max-boot-devices OF property.
+        f = open("/sys/firmware/devicetree/base/ibm,max-boot-devices", "rb")
+        limit = int.from_bytes(f.read(4), byteorder='big')
+        boot_list = boot_list[0:limit]
```

Comment:
Do you need the ``boot_list[0:limit]`` shouldn't it be enough to have just a ``boot_list[limit]``?

Reply:
@jkonecny12 I guess you meant `boot_list[:limit]`? Yes, I changed if you think that's more clear.

Reply:
For me it's more obvious. When the `0` is there I wonder what's the reason and looking for catches.